tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Cap/Sp: get <strong>an url</strong> and we &gt; get <strong>a URL</strong> and we</li><li>Sp. librairy &gt; library</li><li>Suggest to use "to <strong>display</strong> the mean" instead of "to <strong>plot</strong> the mean" - plot makes me think charting</li><li>Add space: "&gt;30°C" to "&gt; 30°C"</li><li>Shorten the note about long wait; suggestion: "note that it may take a moment for the image to load after the cell completes execution"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Done.</p><p> </p>